cdap-docs/admin-manual/source/installation/ambari.rst

```diff
@@ -63,39 +65,46 @@ The **repository version** (shown in the commands below as |literal-cdap-slash-s
-+================+=================+=========================+
-| CDAP 4.1.x     | ``cdap/4.1``    | HDP 2.0 through HDP 2.6 |
-+----------------+-----------------+-------------------------+
-| CDAP 4.0.x     | ``cdap/4.0``    | HDP 2.0 through HDP 2.5 |
-+----------------+-----------------+-------------------------+
-| CDAP 3.6.x     | ``cdap/3.6``    | HDP 2.0 through HDP 2.4 |
-+----------------+-----------------+-------------------------+
-| CDAP 3.5.x     | ``cdap/3.5``    | HDP 2.0 through HDP 2.4 |
-+----------------+-----------------+-------------------------+
-| CDAP 3.4.x     | ``cdap/3.4``    | HDP 2.0 through HDP 2.4 |
-+----------------+-----------------+-------------------------+
-| CDAP 3.3.x     | ``cdap/3.3``    | HDP 2.0 through HDP 2.3 |
-+----------------+-----------------+-------------------------+
-| CDAP 3.2.x     | ``cdap/3.2``    | HDP 2.0 through HDP 2.3 |
-+----------------+-----------------+-------------------------+
-| CDAP 3.1.x     | ``cdap/3.1``    | HDP 2.0 through HDP 2.2 |
-+----------------+-----------------+-------------------------+
-| CDAP 3.0.x     | ``cdap/3.0``    | HDP 2.0 and HDP 2.1     |
-+----------------+-----------------+-------------------------+
++---------------------------------------------------------+
+| Supported Hortonworks Data Platform (HDP) Distributions |
++-------------------------------+-------------------------+
+| CDAP Series or Release        | Hadoop Distributions    |
++===============================+=========================+
+| CDAP 4.1.1                    | HDP 2.0 through HDP 2.6 |
++-------------------------------+-------------------------+
+| CDAP 4.1.0                    | HDP 2.0 through HDP 2.5 |
++-------------------------------+-------------------------+
+| CDAP 4.0.x                    | HDP 2.0 through HDP 2.5 |
++-------------------------------+-------------------------+
+| CDAP 3.6.x                    | HDP 2.0 through HDP 2.4 |
++-------------------------------+-------------------------+
+| CDAP 3.5.x                    | HDP 2.0 through HDP 2.4 |
++-------------------------------+-------------------------+
+| CDAP 3.4.x                    | HDP 2.0 through HDP 2.4 |
++-------------------------------+-------------------------+
+| CDAP 3.3.x                    | HDP 2.0 through HDP 2.3 |
++-------------------------------+-------------------------+
+| CDAP 3.2.x                    | HDP 2.0 through HDP 2.3 |
++-------------------------------+-------------------------+
+| CDAP 3.1.x                    | HDP 2.0 through HDP 2.2 |
++-------------------------------+-------------------------+
+| CDAP 3.0.x                    | HDP 2.0 and HDP 2.1     |
++-------------------------------+-------------------------+
 
 .. _ambari-compatibility-matrix-end:
 
-**Note:** The CDAP Ambari service has been tested on Ambari Server 2.0 and 2.1, as
-supplied from Hortonworks.
+**Notes:**
+
+- The CDAP Ambari service has been tested on Ambari Server 2.0 and 2.1, as
```

Comment:
We have tested this on newer versions. Versions 2.3 - 2.5 are the supported versions that have been tested